grid-client/integration_tests/vm_volume_test.go

```diff
@@ -0,0 +1,89 @@
+package integration
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/threefoldtech/tfgrid-sdk-go/grid-client/deployer"
+	"github.com/threefoldtech/tfgrid-sdk-go/grid-client/workloads"
+)
+
+func TestVMWithVolume(t *testing.T) {
+	tfPluginClient, err := setup()
+	if err != nil {
+		t.Skipf("plugin creation failed: %v", err)
+	}
+
+	publicKey, privateKey, err := GenerateSSHKeyPair()
+	require.NoError(t, err)
+
+	nodes, err := deployer.FilterNodes(
+		context.Background(),
+		tfPluginClient,
+		generateNodeFilter(WithFreeSRU(3)),
```

Comment:
Why sru is 3 and you request 2 only in the next line?

Reply:
I don't really remember but it seems 1 for Volume and 2 for minimum rootfs, the 2 next line should be 1